### ISSUE TITLE
Formatting issues with Rec 42

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/metanorma-unece#2]
1. the Foreword should be before ToC (and not displayed within ToC)
2. the Rec title should not be shown on page 1
3. footnotes should use a smaller font, and be left aligned (not justified)
4. Table one should not have left and right borders,
5. image in the box is going out of the right border, 
6. box text should not be italics 
7. D. 1 is missing (they're off by 1) 
8. "Annex I" should be "Annex I:\nAn overview..." and left aligned and split into two lines with a larger font 
9. Annex sections should be numbered "1,2,3..." instead of "A,B,C..." 
10. the last text of Annex II should be non-bolded "for each indicator mentioned in this table, ..."
11. Tables in Annexes are displayed as "Table A2" instead of "Table II.2"

**Comment by opoudjis** [comment]
2. Fixed.

3. Fixed.

4. The tables in the Annex are formatted completely differently. I am disinclined to start micromanaging table styling: the message needs to be that they are sacrificing styling.

6. Fixed.

7. Section numbering is not dealing well with hanging clauses. I will try to fix it.

9. I'm fixing Annex subsection numbering, but I regard this as a bug: it's an unnecessary inconsistency with main section numbering. And they have a lot of that.

10. I was trying to make it a header cell, because it was shaded, but those are being styled as all-boldface. I do not wish to tinker with table rendering (and these tables have capricious rendering), so I've made it a normal cell.

11. Cannot reproduce.

Remaining, all of which are work: 1, 5, 7, 8.
